### PR TITLE
Use same OCMock as Firebase

### PR DESCRIPTION
**Pull request by paulb777**
Fix https://github.com/firebase/firebase-ios-sdk/issues/9591

### CODE REVIEW

**Comment by ncooke3** [comment]
I think this comment might still be relevant: https://github.com/google/GoogleUtilities/pull/58#issuecomment-923281109

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
     - name: Setup check
       run: |
         brew update
-        brew install clang-format@13
+        brew install clang-format@14
```
Comment (ncooke3) [inline .github/workflows/check.yml:33]: There's also a "brew install clang-format@**13**" mentioned in the `README.md`.

**Review by ncooke3 — APPROVED** [review]
LGTM on green!

**Comment by paulb777** [comment]
Only the relatively common catalyst flake. I'll merge here and publish 7.7.1 for SPM

**Pull request merged**